GitSearch/Models/Constants.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Constants.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/21/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+enum Constants {
+    
+    enum ColorScheme {
+        static let backgroundGrey = #colorLiteral(red: 0.9647058824, green: 0.9647058824, blue: 0.9647058824, alpha: 1)
+        static let borderGrey = #colorLiteral(red: 0.9215686275, green: 0.9215686275, blue: 0.9215686275, alpha: 1)
+        static let mainRed = #colorLiteral(red: 0.9450980392, green: 0.4078431373, blue: 0.3450980392, alpha: 1)
```

Comment:
Probably it's not so good idea to include current color in the constant name. The whole purpose of this constant is to encapsulate this info and to allow its easy tweaking. If, e.g., later you decide to change this color to green, it will either require to rename this constant or otherwise its name become misleading. Something like `mainTint` or `secondaryTint` may be more appropriate.